docs/migrating/from-create-react-app.md

```diff
@@ -39,7 +39,7 @@ Here's an example `package.json`:
 
 ## Static Assets and Compiled Output
 
-Create React App uses the `public` directory for the [entry HTML file](https://create-react-app.dev/docs/using-the-public-folder), whereas Next.js uses it for static assets. It's possible to add static assets here, but Create React App recommends importing them directly from JavaScript files.
+Create React App uses the `public` directory for the [entry HTML file](https://create-react-app.dev/docs/using-the-public-folder). It is located at the project's root. Next.js uses it for static assets. When migrating from Create React App, the location of the `public` directory is the same. It's possible to add static assets in this directory, but Create React App recommends importing them directly from JavaScript files.
 
 - Move any images, fonts, or other static assets to `public`.
 - Convert `index.html` (the entry point of your application) to Next.js. Any `<head>` code should be moved to a [custom `_document.js`](/docs/advanced-features/custom-document.md). Any shared layout between all pages should be moved to a [custom `_app.js`](/docs/advanced-features/custom-app.md).
```

Comment:
Should this say something about converting `public/index.html` to `pages/index.js`?

Reply:
We do that on line `54`.

Reply:
I think line 54 is a bit confusing because its saying "create an index.js file" but CRA already has an index file so I think renaming/converting is better?

Reply:
That's a good point. I'll change the instruction to say that move the `index.js` file to `pages` directory.

Reply:
I forgot how CRA is formatted so I just did `npx create-react-app@latest` and here's what I found:

- CRA `src/index.js` seems to be similar to Next.js `pages/_document.js`
- CRA `src/App.js` seems to similar to Next.js `pages/index.js`
- CRA `public/index.html` seems to be similar to Next.js `pages/_document.js` or perhaps a layout in `pages/_app.js`

The problem I'm finding is that CRA doesn't come with a router so its almost impossible to give clear instructions here since only the user understand their own CRA application.

Reply:
To tackle the problem of routing, I think we do mention that Next.js has a file system based routing and provide a link for the developer to go through the link and also a link for migrating from react-router.

If we want to add conceptual information by explaining how routing works with Next.js and things they might need to go through when migrating, then we can add more information to the guide and provide basic code snippets (or not). I am not sure how Next.js team feels about adding in-depth information in this guide.